packages/fx-core/resource/package.nls.json

```diff
@@ -75,6 +75,7 @@
   "core.progress.generateTemplate.detail": "Downloading and unzipping Template",
   "core.progress.generateSample": "Generating Sample",
   "core.progress.generateSample.detail": "Downloading and unzipping Sample",
+  "core.progress.downloadDirectory.detail": "Downloading sample directory from GitHub",
```

Comment:
Do we need to add new message for this since it is an internal engineering improvement?

Reply:
The original message is "Downloading and unzipping Sample". The new method doesn't involve unzip process so I think it's inappropriate to use original message for the new method.

Reply:
Updated. However, the final message shown will be "Generating sample: Downloading sample" which may be a little strange. I will discuss it with you offline.

Reply:
After discussion, I changed this message to be "Generating sample: Downloading sample ${sampleId}".